File: Blog-web-service/src/main/java/com/blog/WebServiceApplication.java
package com.blog;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;

/**
 * @author JeungNyeongJae
 */

@SpringBootApplication
@EnableEurekaClient
public class WebServiceApplication {

    public static void main(String[] args) {
        SpringApplication.run( WebServiceApplication.class , args );
    }
}

File: Blog-web-service/src/main/java/com/blog/WebServiceApplication.kt
package com.blog

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.netflix.eureka.EnableEurekaClient

/**
 *  @author JeungNyeongJae
 *  @date 2019/11/7
 *
 */
@SpringBootApplication
@EnableEurekaClient
open class WebServiceApplication

fun main(args: Array<String>) {
    runApplication<WebServiceApplication>(*args)
}
